app/components/sortby-dropdown/template.hbs

```diff
@@ -0,0 +1,17 @@
+<span>
+    <div class="dropdown">
+        <button class="btn btn-select dropdown-toggle discover-filter-facets align-left"
+            type="button" id="sortBy" data-toggle="dropdown"
+            aria-haspopup="true" aria-expanded="true">
+            Sort by: <b>{{sortDisplay}}</b>
```

Comment:
This wasn't bolded on preprints. I think it looks a touch better without. Thoughts?